docs/index.rst

```diff
@@ -112,6 +112,7 @@ The OPC UA Advanced tab allows advanced configuration parameters to be set.
     The path does not include the *Objects* folder or the subscribed Node.
   - **Full OPC UA Path meta data name**: The data point name to use when adding the full OPC UA path to every reading. Default is *OPCUAPath*.
   - **Debug Trace File**: Enable the S2OPCUA OPCUA Toolkit trace file for debugging. If enabled, log files will appear in the directory */usr/local/fledge/data/logs*.
+  - **MonitoredItem block size**: The number of MonitoredItem to be added on to the OPCUA server in single call.
```

Comment:
We need to think about how meaningful this is to the user. Some indication of why we might change this in the documentation would be useful.